apps/api/src/index.js

```diff
@@ -43,7 +43,7 @@ app.use(bodyParser());
 app.use(error(formatError));
 
 app.use(serve(path.resolve(__dirname, '../public')));
```

Comment:
So we don't need this anymore, right?

Reply:
Yes we do, because of all the assets: CSS, and stuff (look at the links)